docs/contributing.rst

```diff
@@ -0,0 +1,188 @@
+Contributing to Synapse
+=======================
+
+* `Project Style Guide`_.
+* `Docs Generation`_.
+* `Contribution Process`_.
+
+
+Project Style Guide
+-------------------
+
+The following items should be considered when contributing to Synapse:
+
+* The project is not currently strictly PEP8 compliant.  Compliant sections
+  include the following:
+
+  - `Whitespace in Expressions and Statements <https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements>`_.
+
+* Please keep line lengths under 120 characters.
+* Use single quotes for docstrings, not double quotes.
+* Use Napoleon format when writing docstrings. This format is very readable
+  and will allow type hinting for IDE users. Use the Google format docstring
```

Comment:
Is there any other pydoc instances where google has a style ( ie, do they also have a pydoc style for when a project is *not* using sphinx/napoleon?  It feels to me like it's correct to say the "google dialect" of sphinx/napoleon or some such, but this is the first i've looked into it :D